mote/__init__.py

```diff
@@ -29,7 +29,7 @@
 
 import flask, random, string, json, util, re
 import dateutil.parser, requests, collections
-import logging
+import logging, soke
```

Comment:
Interesting, thanks for clearing that up for me :)
We should probably add the import. I remember someone mentioning that random 500s happened occasionally. Maybe those could have been related to this issue.